doc/sphinx/Makefile.am

```diff
@@ -55,6 +55,11 @@ TAG	?= $(shell [ -n "`git tag --points-at HEAD | head -1`" ]		\
 
 BOOK		= none
 
+DEPS_Clusters_from_Scratch	= shared/pacemaker-intro.rst
+DEPS_Pacemaker_Administration	= shared/pacemaker-intro.rst
```

Comment:
I thought about doing this when converting a previous book, so I'm glad to see it's happening now.